### PR TITLE
HashMap java script

### PR DESCRIPTION
**Pull request by howdyAnkit**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes I have followed the contribution guidelines.)

### Description

([Javascript] HashMap DataStructure was opened by Me itself and was assigned by @iamrajiv .)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(https://github.com/TesseractCoding/NeoAlgo/issues/350.)

### CODE REVIEW

**Comment by howdyAnkit** [comment]
I am aware NeoAlgo is getting spammed with PR. if possible can anyone kindly review my PR?

**Review by mar-galho — APPROVED** [review]

File: JavaScript/ds/HashMap.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+
+const birdCensus = new HashMap(16);
+
+birdCensus.assign('mandarin duck', 'Central Park Pond');
+birdCensus.assign('monk parakeet', 'Brooklyn College');
+birdCensus.assign('horned owl', 'Pelham Bay Park');
+
+console.log(birdCensus.retrieve('mandarin duck'));
+console.log(birdCensus.retrieve('monk parakeet'));
+console.log(birdCensus.retrieve('horned owl'));
```
Comment (atarax665) [inline JavaScript/ds/HashMap.js:180]: Add sample input/output in in the end in commented form.

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Review by HarshCasper — APPROVED** [review]

File: JavaScript/ds/HashMap.js (hunk 1 above)
Comment (howdyAnkit) [reply JavaScript/ds/HashMap.js:180]: Thank you for reviewing Abhinav. Please Check I have made the requested changes

**Review by mar-galho — APPROVED** [review]

**Comment by howdyAnkit** [comment]
@atarax665 can you please review this...?

**Pull request closed without merging**